enhancements/microshift/audit-log-configuration-options.md

```diff
@@ -0,0 +1,255 @@
+
+| OpenShift Audit Policy Profile | Description                                                                                                                                                                                                                           |
+|--------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| None                           | No requests are logged; even OAuth access token requests and OAuth authorize token requests are not logged.                                                                                                                           |
+| Default                        | Logs only metadata for read and write requests; does not log request bodies except for OAuth access token requests. This is the default policy.                                                                                       |
+| WriteRequestBodies             | In addition to logging metadata for all requests, logs request bodies for every write request to the API servers (create, update, patch, delete, deletecollection). This profile has more resource overhead than the Default profile. |
+| AllRequestBodies               | In addition to logging metadata for all requests, logs request bodies for every read and write request to the API servers (get, list, create, update, patch). This profile has the most resource overhead.                            |
+
+MicroShift will reuse the existing OpenShift custom resource, `apiserver.config.openshift.io`, defined as part of the [OpenShift API](https://github.com/openshift/api/blob/master/config/v1/types_apiserver.go) as part of the translation process from "profile" to "policy". Logic for translating the OpenShift profiles to the `policy.audit.k8s.io` objects is provided by the [openshift/library-go](https://github.com/openshift/library-go/tree/release-4.14/pkg/operator/apiserver/audit) library and is intended for import into external code bases.
+
+To enable OpenShift profiles, MicroShift will get the `apiserver.auditLog.profile` value from the MicroShift config and internally wrap this value into an `APIServer.config.openshift.io/v1` API object. This instance will be passed to the [library-go GetAuditPolicy function](https://github.com/openshift/library-go/blob/release-4.14/pkg/operator/apiserver/audit/audit_policies.go#L93), which will return the `policy.audit.k8s.io/v1` equivalent.
```

Comment:
I don't see the place where the field for the profile value is named. Am I just missing it?

Reply:
The API itself is defined in the [openshift/api/config/v1:types_apiserver.go](https://github.com/openshift/api/blob/750a3e21ebaf57f97e022f2c7f5ed784322de844/config/v1/types_apiserver.go#L87).  I'll link this in the doc for better context.